library/src/main/java/org/apache/apex/malhar/lib/window/Tuple.java

```diff
@@ -45,11 +45,12 @@
    */
   class PlainTuple<T> implements Tuple<T>
   {
-    private T value;
```

Comment:
Why change this to final? What if I have a stateless transform in between 2 windowed operator and I want to change the value of the tuple. Then I have to recreate the Tuple instance